frontend/pages/SoftwarePage/SoftwareTitles/SoftwareTable/SoftwareTable.tsx

```diff
@@ -212,8 +213,8 @@ const SoftwareTable = ({
       order_direction: orderDirection,
       order_key: orderKey,
       page: 0, // resets page index
-      ...getSoftwareFilterForQueryKey("allSoftware"), // Reset to all software
-      ...getSoftwareVulnFiltersForQueryKey(vulnFilters),
+      ...buildSoftwareFilterQueryParams("allSoftware"), // Reset to all software
+      ...buildSoftwareVulnFiltersQueryParams(vulnFilters),
```

Comment:
both of these functions are called twice, we could call once each and put in constants instead

Reply:
Nice catch, Just looked into this--One of the two functions (buildSoftwareFilterQueryParams) called twice is using two different arguments so I can't put in constant. I'm going to keep both using the same spread operator so it's clearer to read to have buildSoftwareFilterQueryParams and buildSoftwareVulnFiltersQueryParams written similarly.

Here:
``` 
      ...buildSoftwareFilterQueryParams("allSoftware"), // Reset to all software
      ...buildSoftwareVulnFiltersQueryParams(vulnFilters),
      ```
      
      and the other instance is
      
      ```
      ...buildSoftwareVulnFiltersQueryParams(vulnFilters),
      ...buildSoftwareFilterQueryParams(value),
      ```